ffi/diplomat/tests/completeness.rs

```diff
@@ -0,0 +1,843 @@
+use diplomat_core::*;
+use rustdoc_types::{Crate, Item, ItemEnum};
+use std::collections::BTreeSet;
+use std::fs::File;
+use std::path::PathBuf;
+
+const COVERED_CRATES: &[&str] = &["icu"];
```

Comment:
Discussion: I can see pros and cons of using the meta-crate versus the smaller crates for this. What do you see as the tradeoffs?

Reply:
Personally I like the smaller crates since we can add crates one by one as necessary.

Reply:
Pros and cons of using `icu::` in my view:

- Pro: Everything in `icu::` is automatically tracked by the tool
- Pro: Canonical names in the rustlink
- Con: Creates a dichotomy where some things are tracked by `icu::` and other things by the smaller crates

Reply:
Also e.g. `icu::` doesn't export FixedDecimal, for one, which we have exposed on Diplomat

Reply:
The tool supports multiple crates, you can just run it on ["icu", "fixed_decimal"]